src/test/java/software/amazon/awssdk/crt/test/StringUtilsTest.java

```diff
@@ -10,7 +10,7 @@
 import software.amazon.awssdk.crt.utils.StringUtils;
 
 
-public class StringUtilsTest {
+public class StringUtilsTest extends CrtTestFixture {
```

Comment:
Just a note, mostly to myself: If you use anything that uses native C code, you HAVE to have `extends CrtTestFixture` or otherwise the test will fail!

What is really unfortunate is that it just reports it failed trying to call the native function but it doesn't say why it failed, that it was because the CRT was not loaded, or anything - it just says it fails calling the native function.
Spent WAY to long trying to find out why it was not working, debugging everything from top to bottom, and scratching my head on why it only failed on tests.

Moral of the story: always make sure `CrtTestFixture` is present when testing anything that touches native code. If you get strange errors but only in tests, double check to make sure `CrtTestFixture` is present.